features/users/user_dashboard.feature

```diff
@@ -124,3 +124,18 @@ Feature: User dashboard
     And I should not see "Oldest Series"
   Then I should see "Recent bookmarks"
     And I should see "Work 5" within "#user-bookmarks"
+
+  Scenario: Sorting within user or collection filtered results removes fandom filter
```

Comment:
The scenario should really describe the _desired_ behavior, not the bug, e.g. "You can follow a fandom link from a user's dashboard and then use the filters to sort within that fandom." Also, since this doesn't explicitly test collections, it shouldn't mention collections.